packages/mobile/src/app/App.tsx

```diff
@@ -42,8 +42,16 @@ import ErrorBoundary from './ErrorBoundary'
 import { ThemeProvider } from './ThemeProvider'
 import { AudiusTrpcProvider } from './TrpcProvider'
 
+const navigationIntegration = Sentry.reactNavigationIntegration({
+  enableTimeToInitialDisplay: true
+})
+
 Sentry.init({
-  dsn: env.SENTRY_DSN
+  dsn: env.SENTRY_DSN,
+  integrations: [navigationIntegration, Sentry.reactNativeTracingIntegration()],
+  enableUserInteractionTracing: true,
+  // send 30% of traces to reduce noise
+  tracesSampleRate: 0.3
```

Comment:
reasonable number, I do wonder if we should just have all enabled? Is there any $$ cost or perf cost to bumping this up?

Reply:
nope just caps us at [10M spans](https://sentry.io/pricing/), was a premature optimization, we can bump to one and adjust as needed, updating